roxctl/central/generate/generate.go

```diff
@@ -203,6 +203,15 @@ func createBundle(logger logger.Logger, config renderer.Config) (*zip.Wrapper, e
 func OutputZip(logger logger.Logger, config renderer.Config) error {
 	logger.InfofLn("Generating deployment bundle...")
 
+	if config.EnablePodSecurityPolicies {
+		logger.InfofLn("Central deployment bundle includes PodSecurityPolicies (PSPs). This is incompatible with Kubernetes >= v1.25.")
+		logger.InfofLn("Use --enable-pod-security-policies=false for disabling PodSecurityPolicies.")
+		logger.InfofLn("For the time being PodSecurityPolicies remain enabled by default in deployment bundles and need to be disabled explicitly for Kubernetes >= v1.25.")
```

Comment:
I think this message duplicates the previous line? 🤔

Reply:
tbh, I don't see this as a duplicate. l208 as a (intended!) very short explanation of what is required to be done in order to achieve a specific goal. l209 on the other hand gives 
(1) a more general explanation of the context and
(2) implies the heads-up that this default setting might be up for modification in the future.

If you have a proposal for a different wording, we can of course discuss this. Or ask Gaurauv.

Reply:
Sounds good!